src/config/static/sortedMarkets.ts

```diff
@@ -0,0 +1,27 @@
+export const SORTED_MARKETS = [
```

Comment:
Not quite understand what is it and what should I do when I will add new market? Maybe we can automate this and put into prebuilt script?